### ISSUE TITLE
Replace ClutterBoxLayout with XfdashboardBoxLayout for right-to-left languages

### ISSUE THREAD

**Issue by gmc-holle** [gmc-holle/xfdashboard#66]
It can be seen from the screenshot in issue #64 that ClutterBoxLayout respects RTL. Everything in horizontal layouted container with ClutterBoxLayout will be build from right to left. The documentation of ClutterBoxLayout did not mention it and it seems to be the only layout manager respecting this setting.

So a new layout manager XfdashboardBoxLayout should be created which is mostly exactly the same as ClutterBoxLayout which without respecting RTL so that themes will look the same everywhere. But RTL setting at text-boxes (and labels which are read-only text-boxes) must still respect RTL.

**Comment by GreenLunar** [comment]
I need to send another screenshot.

Maybe `gtk_widget_set_direction` is relevant:
https://developer.gnome.org/gtk3/unstable/GtkWidget.html#gtk-widget-set-direction
http://git.gnome.org/browse/ekiga/commit/?id=76c9b23

**Comment by gmc-holle** [comment]
Clutter is used for graphic display and ClutterActor is the "equivalent" to GtkWidget. ClutterActor provides a similar function called clutter_actor_set_text_direction(). According to the documentation the problem with using this function is that all children will also get the text-direction set.

https://developer.gnome.org/clutter/1.12/ClutterActor.html#clutter-actor-set-text-direction
```
If self implements ClutterContainer then this function will recurse inside all the children of self (including the internal ones). 
```

I understand that if I set LTR at the top-level widget (ClutterStage) all children will be set to LTR - even the text-boxes. But these ones should stay at RTL.

I'll check if it could be solved via a simple change at the themes or if implementing this replacement layout is still needed.

**Comment by gmc-holle** [comment]
Fixed with commit acad499c17ad3d1baf3b987ef37e5dbb67344ffa

Closing issue.
